roles/config-rh-sso/tasks/prereq.yml

```diff
@@ -21,3 +21,8 @@
     immediate: yes
   with_items:
   - "{{ rh_sso_port_list }}"
+
+- name: 'Download PostgreSQL JDBC driver'
+  get_url:
+    url: https://jdbc.postgresql.org/download/postgresql-42.2.16.jar
+    dest: /tmp
```

Comment:
We probably only want this to occur when we `rh_sso_standalone_ha: true` right?

Reply:
That's right... Considering your other comments, I might move it inside the `setup-sso-ha.yml` file, considering that this might not be an actual `pre-req` if we want to use another database in the future.

Reply:
This will also help to better consolidate the dependencies if we want to use another database.